guides/release/configuring-ember/optional-features.md

```diff
@@ -170,3 +170,45 @@ backwards compatibility for existing templates while new template-only
 components gain the advantages of this feature.
 
 For more information, see [RFC #278](https://github.com/emberjs/rfcs/blob/master/text/0278-template-only-components.md).
+
+### default-async-observers
+
+With this feature *enabled*, Ember will run all observers in the application
+asynchronously by default. This leads to observers running in the runloop
+after the one in which the properties it observers were updated.
+
+If the feature is *disabled*, observers run synchronously
+and will then update as soon as their observed properties update.
+
+Async observers are more performant than those that run synchronously
+and can help you to manage your application state in a more predictable manner.
+This is one of the reasons, why the `default-async-observers` feature is
+**enabled by default** in newly created, modern Ember applications.
+
+The `default-async-observers` feature affects the behaviour of observers application-wide,
+but can still set single instances of observers to run synchronously or async
+manually. By using the `sync: true` option, observers who are otherwise async by default,
+can be marked as synchronous manually. Vice versa, otherwise synchronously running observers
+can be set to run asynchronously using the `sync: false` option.
+
+
+```javascript
+import { observer } from '@ember/object';
+
+Person.reopen({
+  partOfNameChanged: observer({
+  dependentKeys: ['firstName', 'lastName'],
+  function() {
```

Comment:
Is the code example correct here, or should it rather be

```suggestion
  fn() {
```

I couldn't find any documentation on it (which seems to be the reason of this PR 😅 )

Reply:
It is indeed supposed to be named `fn`

Also, the RFC has a bit of detail on the exact signature too https://github.com/emberjs/rfcs/blob/master/text/0494-async-observers.md

Reply:
Super, thank you for the reference 🙇‍♀